src/box/iproto.cc

```diff
@@ -1325,24 +1346,19 @@ iproto_connection_on_output(ev_loop *loop, struct ev_io *watcher,
 {
 	struct iproto_connection *con = (struct iproto_connection *) watcher->data;
 
-	try {
-		int rc;
-		while ((rc = iproto_flush(con)) <= 0) {
-			if (rc != 0) {
-				ev_io_start(loop, &con->output);
-				return;
-			}
-			if (! ev_is_active(&con->input) &&
-			    rlist_empty(&con->in_stop_list)) {
-				ev_feed_event(loop, &con->input, EV_READ);
-			}
+	int rc;
+	while ((rc = iproto_flush(con)) <= 0) {
+		if (rc != 0) {
+			ev_io_start(loop, &con->output);
+			return;
+		}
+		if (! ev_is_active(&con->input) &&
```

Comment:
In new code we do not put whitespace after unary operators.

Reply:
fixed, thanks